### DISCUSSION TITLE
Ability to publish weblink field URL's to data aggregators or show in query results

### DISCUSSION THREAD

**Discussion by acbentley** [specify/specify7#5936] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
For data publishing and query results, it would be useful and highly desirable to be able to publish the full URL from a weblink field including the base URL and the field value.  Currently, one can only publish or show the value in the field which negates the utility of a weblink field for publishing to aggregators through DwC or allowing linkage through from a query result.

**Describe the solution you'd like**
Ideally, it would be good to be able to somehow format the field to include the base URL

**Describe alternatives you've considered**
One alternative would be to allow attachments to be URLs rather than physical resources that would allow users to include external web resources/links as attachments.  This would help with queries composed for DwC mapping and allow for a larger range of attachment types to be used by collection users.

**Reported By**
Biodiversity Institute

**Additional context**
I thought that Tim had found a way to make this possible in Specify 6 but have no further details on how or if he managed that.

**Comment by PhatWheZ** [comment]
Hi guys, perhaps a solution would be to have an index list that is compiled via the Specify database being publicly exposed. Then link that searchable index list to unique URLs which proxy through the asset-server.

**Comment by maxpatiiuk** [comment]
Related to (or maybe even a duplicate of) https://github.com/specify/specify7/issues/2814
